### DISCUSSION TITLE
why is my UTM in chinese (i'm french by the way)

### DISCUSSION THREAD

**Discussion by ssjblue-xxx** [utmapp/UTM#3741] [UTM Help (macOS)]
idk why but my utm is chinese is there a way to make it in french or english ?

**Comment by osy** [comment]
I don’t think there is a French translation. The way your system is set up, it probably defaulted to Chinese. See https://support.apple.com/guide/mac-help/change-the-system-language-mh26684/mac for details on adding multiple languages to your Mac.
